### ISSUE TITLE
Delay JavaScript Execution compatibility with the Eikra theme

### ISSUE THREAD

**Issue by DahmaniAdame** [wp-media/wp-rocket#4606]
**Before submitting an issue please check that you’ve completed the following steps:**
- [x] Made sure you’re on the latest version
- [x] Used the search feature to ensure that the bug hasn’t been reported before

**Describe the bug**
When the Eikra is used with the Delay JavaScript Execution, the preload is not working even after user interaction. 

No console errors show up. 

Excluding the following fixes the issue: 

```
/jquery(-migrate)?((.min|.slim|.slim.min)|.*?([0-9]+)(.min|.slim|.slim.min))?.js
/eikra/assets/js/
EikraObj
```

Notice that I had to use a different jQuery pattern because of this issue - https://github.com/wp-media/wp-rocket/issues/4607

**To Reproduce**
1. Use the Eikra
2. Enable the page preloaded (not our cache preloader)
3. See error

**Expected behavior**
The preloader should work with no issues at least after user interaction.

**Screenshots**
N/A

**Additional context**
Ticket - https://secure.helpscout.net/conversation/1732517182/315197/

**Backlog Grooming (for WP Media dev team use only)**
- [ ] Reproduce the problem
- [ ] Identify the root cause
- [ ] Scope a solution
- [ ] Estimate the effort

**Comment by piotrbak** [comment]
@DahmaniAdame Is `Page Preloader` a feature of this theme?

If so, is everything working correctly when it's disabled and Delay JS is enabled?

**Comment by DahmaniAdame** [comment]
Yes. It's a feature of the theme that can be turned on and off. 

> If so, is everything working correctly when it's disabled and Delay JS is enabled?

For memory, it worked with no issues when the preloader was turned off.

I asked the customer to give me access to the site to confirm that part. 

I will keep you informed.

**Comment by DahmaniAdame** [comment]
@piotrbak I did some tests. There is no issue if the `Page Preloader` is deactivated on the theme when Delay JavaScript Execution is activated.

**Comment by piotrbak** [comment]
Will be fixed in the 3.13 backend

**Comment by piotrbak** [comment]
Handled in 3.13 using dynamic lists
